ir_datasets/etc/downloads.json

```diff
@@ -1,4063 +1,4065 @@
 {
```

Comment:
looks like the indentation of this file was changed. Can you revert to the original and make the change without modifying the indentation of the whole file?